docs/3.x.x/getting-started/install-requirements.md

```diff
@@ -0,0 +1,197 @@
+# Installation Requirements
+
+### 👋 Welcome onboard!
+
+Welcome to the open source [headless CMS](https://strapi.io) developers love. Our users love Strapi because it is open source, MIT licensed, fully customizable and based on nodejs. Strapi lets you manage your content and distribute it anywhere.
+
+This page covers installing the [basic installation requirements for Strapi](#basic-installation-requirements).
```

Comment:
Yes you're right, sort of. I kept the link like the original version because it goes to a section lower on the page and is good for SEO. On this point, I think it's a good idea to keep it in.

Reply:
People might actually be confused, so I removed the link.